pages/common/openssl-ts.md

```diff
@@ -0,0 +1,20 @@
+# openssl ts
+
+> OpenSSL command to generate and verify timestamps.
+> More information: <https://www.openssl.org/docs/manmaster/man1/openssl-ts.html>.
+
+- Generate sha512 timestamp request of `file` (output to `file.tsq`):
+
+`openssl ts -query -data {{file}}  -sha512  -out {{file.tsq}}`
```

Comment:
```suggestion
`openssl ts -query -data {{path/to/file}}  -sha512  -out {{path/to/file.tsq}}`
```

Reply:
Why do some tldr pages (eg openssl, gpg) use simple filenames like {{public.gpg}}, whereas others use {{path/to/file}}? Is there a convention based on the context?

Reply:
Thank you so much for reviewing my pull request ^_^